packages/drawer/src/Drawer.tsx

```diff
@@ -45,7 +46,13 @@ type DrawerProps = {
   hideCancel?: boolean;
 };
 
-const Drawer = ({
+const Drawer = (props: DrawerProps) => (
+  <Portal>
+    <DrawerContainer {...props} />
+  </Portal>
+);
```

Comment:
rendering the portal separately from the drawer contents fixed the issue of the broken focus trap